imgur-screenshot.sh

```diff
@@ -103,6 +117,49 @@ if [ "${1}" = "--check" ]; then
   exit 0
 fi
 
+# Install ubuntu application launcher icon for this user
+if [ "${1}" = "--install-ubuntu" ]; then
```

Comment:
This should go in a separate function and the arg handling down to the bottom where the rest of the args are handled.

The `imgur-screenshot.desktop` file could also be a separate file in the repository.

Reply:
I'd also vouch for renaming this to `--install-desktop` because the [XDG Desktop Entry spec](https://www.freedesktop.org/wiki/Specifications/desktop-entry-spec/) is not specific to Ubuntu.

Reply:
#1 ok

#2 Do you know if Macs follow the XDG spec for icons? I was having a hard time finding info on that and didn't want to conflate linux/mac use cases, so I branded for only the use cases I'd tested.

Reply:
macOS doesn't use any of XDG specs, so this wouldn't work on mac.